### DISCUSSION TITLE
Show contour lines (heights)?

### DISCUSSION THREAD

**Discussion by datenbank-projekt** [andreynovikov/trekarta#189] [General]
Hi,

Is there a way to show contour lines (mountain heights) in Trekarta? I am not able to see them clearly, but navigating in the outdoors would help this a lot.
Wikiloc does show those - but is missing many of the roads and POIs.

**Comment by andreynovikov** [comment]
It shows contour lines if you didn't disable it in map features:

![](https://trekarta.info/images/hiking.png)
